cli/tools/vendor/build.rs

```diff
@@ -1117,7 +1117,8 @@ mod test {
       .unwrap();
 
     assert_eq!(
-      err.to_string(),
+      std::str::from_utf8(&strip_ansi_escapes::strip(err.to_string()).unwrap())
+        .unwrap(),
```

Comment:
I'm a total rust newbie and this line feels a bit icky to me. Conceptually, I think it's the right approach to strip the ansi codes here for testing, but the conversion + unwrapping feels a bit much. Not sure what the proper rust way to do this would be.

Reply:
There is a helper function in the Deno repo that you could use I think. You would need to import `strip_ansi_codes` from `test_util`, see https://github.com/denoland/deno/blob/main/cli/tests/integration/coverage_tests.rs#L264 for an example.

Reply:
Ohh this is great, thanks for sharing! That's a much better solution.